bin/create-release.sh

```diff
@@ -102,6 +107,23 @@ done
 # to older ones
 rm -rf ${ROOT}/${release_path}/dev_releases
 
+if find "${ROOT}/${release_path}/blobs/" -type l >/dev/null 2>/dev/null; then
+  # Migrating from BOSH CLI v1 (ruby) to BOSH CLI v2 (golang)
+  docker run \
+    --interactive \
```

Comment:
Why do we need `--interactive`?

Reply:
Umm. We probably don't, I just copied it from the chunk below. Let me change both and see what falls over.

Reply:
Didn't seem to affect anything, dropped in ecaa4b61e3220c4f44a077437f130bb2ca77d733